docs/user_quick_start.md

```diff
@@ -9,7 +9,7 @@
 
 ## Installation
 
-Run the following (ideally inside a Python virtual environment):
+Run the following (ideally inside a Python virtual environment)::
```

Comment:
This extra colon needs to be removed, and then this can be approved!

Reply:
thanks. Done